metadata.json

```diff
@@ -72,5 +66,13 @@
         "16"
       ]
     }
-  ]
+  ],
+  "requirements": [
+    {
+      "name": "puppet",
+      "version_requirement": ">= 4.0.0 < 6.0.0"
+    }
+  ],
+  "template-url": "file:///opt/puppetlabs/pdk/share/cache/pdk-templates.git",
```

Comment:
Maybe we should change this to `https://github.com/puppetlabs/pdk-templates`

Reply:
Done